### DISCUSSION TITLE
Usage of F macro in Serial.print in Pico?

### DISCUSSION THREAD

**Discussion by martinerk0** [earlephilhower/arduino-pico#1574] [General]
Is using of F macro in `Serial.print(F("looong string"));` situations _useful in Pico_? UNO3 has 2kB of SRAM, but Pico has 264KB of SRAM and flash is supposed to be slower, although according to [this](https://stackoverflow.com/questions/14728968/speed-comparison-eeprom-flash-sram,) not so much. On the other hand, you could [wear out flash](https://electronics.stackexchange.com/questions/67038/what-are-the-implications-of-using-progmem)?

**Comment by earlephilhower** [comment]
No, it's not useful and is a noop.  Constant strings are left in flash and not copied into SRAM.
